src/commands/cli/schemas/compare.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as os from 'os';
+import { FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { fs, Messages, SfdxError } from '@salesforce/core';
+import cli from 'cli-ux';
+import { SchemaUtils } from './collect';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'cli.schemas.compare');
+
+export type Results = {
+  [key: string]: {
+    correspondingFile: string;
+    matches: boolean;
+    reason?: string;
+  };
+};
+
+export default class Compare extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly flagsConfig: FlagsConfig = {};
+
+  public async run(): Promise<Results> {
+    // The "existing schema" is the schema that is stored at the CLI level
+    const existing = await SchemaUtils.getLatestSchemaFiles();
+    // The "latest schema" is the schema that is found in the node_modules
+    const latest = await SchemaUtils.getExistingSchemaFiles();
```

Comment:
Are these meant to be switched?
```suggestion
    // The "existing schema" is the schema that is stored at the CLI level
    const existing = await SchemaUtils.getExistingSchemaFiles();
    // The "latest schema" is the schema that is found in the node_modules
    const latest = await SchemaUtils.getLatestSchemaFiles();
```